### PR TITLE
Add Gitee provider

### PR DESCRIPTION
**Pull request by mowangjuanzi**

### CODE REVIEW

File: src/Gitee/composer.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "socialiteproviders/gitee",
+    "description": "Gitee OAuth2 Provider for Laravel Socialite",
+    "license": "MIT",
+    "authors": [{
+        "name": "mowangjuanzi",
+        "email": "baoguoxiao0538@hotmail.com"
+    }],
+    "require": {
+        "php": "^5.6 || ^7.0",
```
Comment (atymic) [inline src/Gitee/composer.json:10]: can drop < 7.0

Comment (mowangjuanzi) [reply src/Gitee/composer.json]: ok. Let me check it out first

Comment (mowangjuanzi) [reply src/Gitee/composer.json]: https://github.com/SocialiteProviders/Documentation/pull/104

**Comment by atymic** [comment]
https://packagist.org/packages/socialiteproviders/gitee

**Comment by mowangjuanzi** [comment]
thank

**Pull request merged**